python/ray/rllib/models/pytorch/model.py

```diff
@@ -39,7 +39,7 @@ def __init__(self, in_channels, out_channels, kernel, stride, padding,
         conv = nn.Conv2d(in_channels, out_channels, kernel, stride)
         if initializer:
             initializer(conv.weight)
-        nn.init.constant(conv.bias, bias_init)
+        nn.init.constant_(conv.bias, bias_init)
```

Comment:
nn.init.constant is deprecated in favor of the underscore version, hence the change